releasenotes/notes/deprecate_func-25c04f94964cffb2.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    The decorator `requires_qe_access` was deprecated in qiskit-terra 0.17 (released on April 2021)
+    and now is it being removed. The decorator :func:`.online_test` is a one-to-one replacement of it.
```

Comment:
tbh, the whole of `qiskit.test` should probably be private to Terra and not part of the public API.  It's weird that other projects use our test internals that are really meant only for Terra.

`online_test` is only used for one test that shouldn't be part of Terra anyway.

(also: two backticks around `requires_qe_access`)

Reply:
I agree we should have done this a long time ago: https://github.com/Qiskit/qiskit/issues/6862

Reply:
true... force of habit. removed in a2ba0ce89e9abc30ef746022b3458204c0812294